style-sheets/carbon-components-overrides.scss

```diff
@@ -10,6 +12,14 @@
   $button-border-radius: 0.25rem
 );
 
+@use '@carbon/styles/scss/grid';
+
+@include grid.flex-grid();
+
+body {
+  font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
```

Comment:
I won't recommend we do any hardcode style here, so another way to update the font family is to remove it from this file, and make the change in your project. So in your project, import the font family design token and make the changes you want.

Reply:
On that note can we change the single quote from `‘` `’` to `'`?
![image](https://github.com/bcgov/nr-fsa-theme/assets/25162561/2142f652-58fe-4204-a925-2f7f4683ba20)

Reply:
Thanks, Craig!

Reply:
Hi @MCatherine1994 

As per our agreement, this part will be part of the customs styles as the given path for the fonts doesn't work for all teams